lib/bots/syncbot.ts

```diff
@@ -19,7 +19,7 @@ import * as _ from 'lodash';
 import { ProcBot } from '../framework/procbot';
 import { MessengerService } from '../services/messenger';
 import {
-	BasicMessageInformation, CreateThreadResponse, FlowDefinition,
+	BasicMessageInformation, CreateThreadResponse, DirectedEdgeDefinition, FlowDefinition,
```

Comment:
NIT: Splitting this import up so there is one import per line would make this a lot more readable